### ISSUE TITLE
Confusion between likelihood and log likelihood

### ISSUE THREAD

**Issue by noashin** [LoLab-VU/PyDREAM#23]
One of the parameters of *pydream.core.run_dream* is called "likelihood" and the documentation specify it to be **"A user-defined likelihood function"**.

It seems though that the code is actually expecting a log-likelihood function, as seen in *pydream.model.total_logp* line 30:
```
loglike = self.likelihood(q0)
```
This can be easily solved by changing the parameter name to be log_likelihood and changing the documentation accordingly.

**Comment by ortega2247** [comment]
You are right. Thanks for pointing it out. We will be updating the parameter name and the documentation accordingly.

Best,
Oscar
